### DISCUSSION TITLE
Abortable API requests

### DISCUSSION THREAD

**Discussion by sfiannaca** [Mobile-Artificial-Intelligence/maid#228] [Ideas]
Hi @danemadsen this commit serves as a demonstration of utilizing the client to facilitate abortable API requests. 
Although I'm aware you've already refactored those files in the same time lol

https://github.com/sfiannaca/Maid/commit/3463bcc737ab798f0d0ff11b3e297aebcec51483

**Comment by danemadsen** [comment]
Looks good, however im going to be moving away from using my own custom implementations of the API's in favor of langchain
